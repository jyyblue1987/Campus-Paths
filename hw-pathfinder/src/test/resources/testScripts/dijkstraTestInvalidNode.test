# Find a path with Dijkstra's algorithm when there is an invalid node

CreateGraph graph

AddNode graph A
AddNode graph B
AddNode graph C

AddEdge graph A C 1.0
AddEdge graph A D 1.0
AddEdge graph A B 1.0

FindPath graph A D