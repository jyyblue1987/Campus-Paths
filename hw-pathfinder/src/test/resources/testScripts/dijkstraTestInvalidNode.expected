# Find a path with Dijkstra's algorithm when there is an invalid node

created graph graph

added node A to graph
added node B to graph
added node C to graph

added edge 1.000 from A to C in graph
added edge 1.000 from A to D in graph
added edge 1.000 from A to B in graph

unknown: D